arangod/RestHandler/RestBatchHandler.h

```diff
@@ -67,7 +67,7 @@ class RestBatchHandler : public RestVocbaseBaseHandler {
   char const* name() const override final { return "RestBatchHandler"; }
   // be pessimistic about what this handler does... it may invoke V8
   // or not, but as we don't know where, we need to assume it
-  RequestLane lane() const override final { return RequestLane::CLIENT_FAST; }
+  RequestLane lane() const override final { return RequestLane::CLIENT_SLOW; }
```

Comment:
There seems to have been a bug with this Lane (https://github.com/arangodb/arangodb/pull/6196)
Which somehow made this Handle use a faster lane.
Have we made sure we do not reintroduce the issue caused there?

Reply:
(I personally think this handler should be Client_slow just want to make sure we not reintegrate regressions)

Reply:
Yes, since it can take arbitrary requests we should keep it in the slow lane.

Reply:
It's unclear to me why the changes in #6196 (using proper per-request lanes for the child requests) was reverted, but at least for the parent handler we should keep it in the slow lane.